### ISSUE TITLE
Function Request: 

### ISSUE THREAD

**Issue by klm-o** [Asaayu/integrated-voice-control-system#3]
is it possible to support the german language as well?

**Comment by klm-o** [comment]
i could also provide some suggestions and translations for commands that correspond to the german milsim community.

**Comment by Asaayu** [comment]
Not at this point in time, supporting a new language would not only require a direct translation of all the words but also require a native speaker to go through and setup each command with the proper sentence structure and grammar used in German.
This would be a monumental task and even then as more commands are added more work would need to be done.

All this work would be for a very small subset of users, ~4% of Steam users speak German compared to ~40% of Steam users speaking English.

In-regards to starting the work yourself, I'd rather there be no other languages at all then half-done or poorly done versions, especially since that would then create a precedent for others to ask for their language to be included as well, most of them probably aren't even supported by the speech recognition software anyway.

TL:DR: To much work, not enough end users, would never be 100% complete. Not worth putting in the time at the moment unfortunately.
